src/app/components/home/home.component.html

```diff
@@ -17,4 +18,19 @@
       </x-box>
     </x-tab>
   </x-tabs>
-  
\ No newline at end of file
+
+  <gridster [options]="options" class="grid-container">
+      <gridster-item [item]="item" *ngFor="let item of dashboard">
+          <div class="gridster-item-content">
+              UnDragable text
+          </div>
+          <div class="item-buttons">
+            <button md-icon-button md-raised-button class="remove-button btn btn-primary" (click)="removeItem($event, item)"
```

Comment:
I think u should use the Xel button (https://xel-toolkit.org/elements/x-button) to keep the styling throughout the app consistent. You can find the available icons here: https://material.io/icons/